API/urlfrontier.proto

```diff
@@ -306,3 +312,15 @@ message URLStatusRequest {
     string crawlID = 3;
 }
 
+message ListUrlParams {
+  // position of the first result in the list; defaults to 0
+  uint32 start = 1;
+  // max number of values; defaults to 100
+  uint32 size = 2;
+  /** ID for the queue **/
+  string key = 3;
+  // crawl ID
+  string crawlID = 4;
+  // only for the current local instance
+  bool local = 5;
```

Comment:
have you left it in prevision of implementing it in the future?

Reply:
Yes, that was the idea